docs/Getting Started.md

```diff
@@ -44,37 +44,71 @@ Next, make sure everything is set up correctly with a quick test run.
 
-Running this simulation requires substantial computational resources and precise 
-configuration, challenges that have previously slowed down your project.
+With this first example, you learn that running simulations via Inductiva API is
+not much different from running them in your local machine. That's the magic
+of it all! 
 
-With Inductiva, you have the opportunity to run the dam break simulation with 
-your existing configuration files.
+Running a simulation with API involves preparing a Python script with the following
+steps:
+1. Preparing all the configuration files for the simulation in a single input folder;
+2. Instantiate a simulator object that identifies the simulator you want to use.
+In this case, we are going to instantiate the REEF3D simulator object;
+3. Launch the simulation with the `run` method of the simulator object and pass
+a reference to the input folder. This folder will be uploaded to a user's
+remote storage and used in the worker machine that will run the simulation;
+4. Wait for the simulation to finish and download the results to your local machine.
+
+To make it simpler, in this example we will download the input folder with all
+the configuration files necessary to run the dam break simulation.
 
 Here's how:
 
 ```python
 import inductiva
 
-# Download the configuration files for a REEF3D simulation
+# 1 - Download the configuration files for a REEF3D simulation. This folder
+# contains the control.txt and ctrl.txt files that configure the parameters of
+# the mesh and the simulation, respectively.
 input_dir = inductiva.utils.download_from_url(
     "https://storage.googleapis.com/inductiva-api-demo-files/"
     "reef3d-dambreak-dir.zip", unzip=True)
 
-# Initialize the REEF3D simulator object
+# 2 - Initialize the REEF3D simulator object
 simulator = inductiva.simulators.REEF3D()
 
-# Launch the simulation with the downloaded configuration files
+# 3 - Launch the simulation with the downloaded input folder. This will return
+# a task object that can be used to monitor the task and download the outputs.
 task = simulator.run(input_dir=input_dir)
 
-# Wait for the simulation to finish and download the outputs
+# 4 - Wait for the simulation to finish and download the outputs to a default
+# folder in your local machine named `example_simulation`.
 task.wait()
-task.download_outputs()
+task.download_outputs(output_dir="example_simulation")
+```
+
+As the simulation runs information, you will receive information about its progress. When it finishes, you should see the following folder on your local machine:
+
+```bash
+inductiva_output/example_simulation
+|
+|- DIVEMesh_Decomp
+|- DIVEMesh_Log
+|- DIVEMesh_Paraview 
+|- REEF3D_CFD_VTU
+|- REEF3D_Log
+|- control.txt
+|- ctrl.txt
+|- grid-000001.dat
+|- grid-000002.dat
+|- stderr.txt
+|- stdout.txt
 ```
 
+In case you want to analyze the results of the simulation, you can use the
+Paraview software.
+
```

Comment:
While using the Inductiva API, keep in mind that it does not provide built-in 
visualization tools for the simulators. For such complex 3D visualizations, our team recommends open-source tools 
like [ParaView](https://www.paraview.org/download/) or [VTK](https://vtk.org/download/).

Something like this?

Reply:
Made your words mine :D

Reply:
I would avoid the negative tone. Why would you have to say that you don't have something you are actually not supposed to have :)
Why not simply say that:

Now, the all the data produced by the simulator has been transfered to your local machine where you can post-process it and, in some cases, even visualize it. For example, you can visualize the contents of file X and Y using standard visualization tools suchs as A and B.